ui/ducks/swaps/swaps.js

```diff
@@ -986,14 +986,14 @@ export const signAndSendSwapsSmartTransaction = ({
           sourceTokenSymbol,
           swapMetaData,
           swapTokenValue,
-          type: TRANSACTION_TYPES.SWAP,
+          type: TransactionType.swap,
         }),
       );
       if (approvalTxUuid) {
         await dispatch(
           updateSmartTransaction(approvalTxUuid, {
             origin: ORIGIN_METAMASK,
-            type: TRANSACTION_TYPES.SWAP_APPROVAL,
+            type: TransactionType.swap_APPROVAL,
```

Comment:
in `shared/constants/transaction.ts` this is correctly typed `swapApproval`

Reply:
Goooood catch.